apps/microtvm/arduino/template_project/launch_microtvm_api_server.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+function show_usage() {
+    cat <<EOF
+This script is for running microtvm_api_server.
+Usage: launch_microtvm_api_server.sh <microtvm_api_server.py> --read-fd <READ_FD_PATH> --write-fd <WRITE_FD_PATH>
+EOF
+}
+
+if [ "$#" -lt 5 -o "$1" == "--help" ]; then
+    show_usage
+    exit -1
+fi
+
+ARDUINO_VENV_PATH=${HOME}/.tvm/micro_arduino
```

Comment:
Why Arduino code is being touched in this PR which is related to a Zephyr / Zephyr SDK update? I think it would be better to summit Arduino changes on separate PR.

Reply:
Initially I didn't plan for this, but since we are testing both in the same image and the new image has different Python env I had to make this changes. Would it be okay to update the PR title and description to cover all these changes?

Reply:
Ok, makes sense. I think it's ok if you just explain it in the description and leave the title as is.

Reply:
ah, and why it's necessary to remove the `tests/micro/project_api/test_arduino_microtvm_api_server.py` ? Is that right? That test is going away? For what reason?

Reply:
This test file was fully based on importing the project api server to TVM which it doesn't work now because of dependencies. Also I checked with @guberti who wrote this test and it doesn't seem to be necessary to keep it anymore.